test/google_scraping/dashboard/keyword_parser_test.exs

```diff
@@ -17,7 +17,7 @@ defmodule GoogleScraping.Dashboard.KeywordParserTest do
                  total_urls_count: 76,
                  non_ad_results_count: 9,
                  non_ad_results_urls_count: 9
-               } = parsed_results
+               } == parsed_results
```

Comment:
Minor comment, but we should have the expected value on the right side
```elixir
assert parsed_results == %{...}
```

Reply:
Thanks! It was fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/51/commits/6d6cb2653bd5190bac2388cf254c311b64bdad60